lib/std/array_list.zig

```diff
@@ -16,6 +16,10 @@ pub fn ArrayList(comptime T: type) type {
     return ArrayListAligned(T, null);
 }
 
+/// A contiguous, growable list of byte-aligned items in memory.
```

Comment:
"byte-aligned" means align(1) but here we are passing alignment as a comptime parameter

Reply:
Changing to "arbitrarily aligned" in lieu of better phrasing. Thoughts?